frontend/src/components/HomePage.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import Header from '@/components/Header.vue'
+import BetaversionLabel from '@/components/betaversion-label.vue'
+
+const emit = defineEmits(['acceptDisclaimer'])
+
+</script>
+
+<template>
+    <Header />
+    <main>
+      <div class="rvo-progress-tracker">
+      </div>
+      <div class="rvo-form">
+        <div class="flex items-center rvo-layout-column rvo-layout-gap--xl">
+          <div class="flex justify-center rvo-form-intro inline-flex py-5">
+            <h2 class="flex justify-center utrecht-heading-2">Beslisboom AI-verordering</h2>
+            <BetaversionLabel />
```

Comment:
Optional layout consideration: maybe not put the beta version label next to the headers, but put it somewhere higher, for example above the horizontal seperator on the same level as "terug naar het begin"

Reply:
Yes i like that idea :)

Reply:
hmm actually i am reverting it, as we would incorporate this change into the AlgoritmeKader then we will not use the header  because of the logo but still want to have the beta version at every step